backend/models.py

```diff
@@ -12,7 +12,7 @@ class Agent(models.Model):
         max_length=100, unique=True, help_text="Human-readable name for the agent."
     )
     # Local path to agent definition root
-    definition_path = models.FileField(upload_to=None, max_length=100)
+    definition_path = models.FileField(upload_to="files/agents", max_length=100)

```

Comment:
i dont know how high i was when i wrote these model definitions but there should not be a `max_length` on these lmao